<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInsert</class>
 <widget class="QWidget" name="QInsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>插入试题</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>300</width>
     <height>263</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>试题图片:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_SelectPic">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_DataPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>试题难度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="comb_difficulty">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>试题章节:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comb_Chapter">
        <item>
         <property name="text">
          <string>必修1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>必修2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>必修3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>必修4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>必修5</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>试题类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comb_belong">
        <item>
         <property name="text">
          <string>选择题</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>填空题</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>问答题</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>选做题</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lab_Answer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>答案:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comb_Answer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btn_SelectAns">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_Answer">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Insert">
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
